### PR TITLE
Get instance id for cookie from v2 env var

### PR DESCRIPTION
**Pull request by aditya-ranshinge**
Signed-off-by: Aditya Ranshinge <aranshinge@rocketsoftware.com>

## Proposed changes

Get instance id for cookie from v2 env var ZWE_zowe_cookieIdentifier

This PR depends upon the following PRs:
https://github.com/zowe/zowe-install-packaging/pull/2787

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [x] If the changes in this PR are meant for the next release / mainline, this PR targets the "staging" branch.
- [x] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zlux/blob/master/CONTRIBUTING.md))
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made
- [ ] Relevant update to CHANGELOG.md
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

### CODE REVIEW

File: plugins/sso-auth/lib/zssHandler.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class ZssHandler {
     this.logger = context.logger;
     this.instanceID = serverConf.instanceID;
     this.sessionExpirationMS = DEFAULT_EXPIRATION_MS; //ahead of time assumption of unconfigurable zss session length
-    const zoweInstanceId = process.env['ZOWE_INSTANCE'];
+    const zoweInstanceId = process.env['ZWE_zowe_cookieIdentifier'];
```
Comment (1000TurquoisePogs) [inline plugins/sso-auth/lib/zssHandler.js]: please do serverConf.cookieIdentifier
and export ZWED_cookieIdentifier=ZWE_zowe_cookieIdentifier in convert-env in zlux-app-server.
Basically I want to see 0 occurrences of process.env in zlux-server-framework, ideally. It makes it much harder to track down issues like the one we see here. This is a WIP, we'll have even better cleanup later.

File: lib/util.js [hunk 2]
```diff
@@ -452,7 +452,7 @@ module.exports.serverSwaggerPluginId = 'org.zowe.zlux'
 module.exports.agentSwaggerPluginId = 'org.zowe.zlux.agent';
 
 module.exports.getCookieName = (port) => {
-  const zoweInstanceId = process.env['ZOWE_INSTANCE'];
+  const zoweInstanceId = process.env['ZWE_zowe_cookieIdentifier'];
```
Comment (1000TurquoisePogs) [inline lib/util.js]: This needs to be changed.
get rid of the isHaMode conditional.
Instead just, cookieName='connect.sid.'+cookieIdentifier
which means you need to change the funciton arg to pass in cookieIdentifier.

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]

File: plugins/sso-auth/lib/zssHandler.js (hunk 1 above)
Comment (aditya-ranshinge) [reply plugins/sso-auth/lib/zssHandler.js]: resolved

File: lib/util.js (hunk 2 above)
Comment (aditya-ranshinge) [reply lib/util.js]: resolved

**Comment by aditya-ranshinge** [comment]
@1000TurquoisePogs I have fixed review comments.

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**